### ISSUE TITLE
Adding gems for capistrano and capistrano-rails

### ISSUE THREAD

**Issue by tenzan** [capistrano/rails#156]
It seems,  those gems should be added in the `:development` group, but you didn't mean so in your `README`.

```
gem 'capistrano', '~> 3.1'
gem 'capistrano-rails', '~> 1.1'
```

Please correct me, if I'm wrong.

**Comment by mattbrictson** [comment]
Yes, that is a good suggestion. Can you submit a PR?

**Comment by tenzan** [comment]
Here you go https://github.com/capistrano/rails/pull/157

**Comment by tenzan** [comment]
https://github.com/capistrano/rails/pull/159

**Comment by mattbrictson** [comment]
Closed via #159.
